idl/chromadb/proto/coordinator.proto

```diff
@@ -96,6 +96,9 @@ message CreateCollectionRequest {
   optional bool get_or_create = 6;
   string tenant = 7;
   string database = 8;
+  // When segments are set, then the collection and segments will be created as
+  // a single atomic operation.
+  repeated Segment segments = 9;  // Optional.
```

Comment:
I understand the case when old client (i.e. FE) (without this field) calls new sysdb service. In that case this repeated field will be empty and thus the sysdb will only create the collection and not the segments. 

What happens in case of the reverse i.e. new client setting this field and issuing a create collections call. My understanding is that the old sysdb server will ignore this field and create only the collection without creating the segments?